scripts/jsdeps_build_hook.py

```diff
@@ -0,0 +1,9 @@
+import subprocess
+
+from hatchling.builders.hooks.plugin.interface import BuildHookInterface
+
+
+class JSDepsBuildHook(BuildHookInterface):
+    def initialize(self, version, build_data):
```

Comment:
I wonder if hatch guarantees that the current working dir is the repo root at this point. If not, this will do the wrong thing. It would be nice to try something like this:

- make new dir somewhere
- clone fontra repo here
- set up Python venv here
- then `pip install ./fontra/`

If that all works, we're good.

Reply:
If not, we may want to specify the repo root dir explicitly through the `cwd` keyword arg of `subprocess.check_output()`.

Reply:
I tried this and it seems to work correctly, so no further work is needed here.